player/lua/osc.lua

```diff
@@ -117,6 +133,7 @@ local state = {
     showhide_enabled = false,
     dmx_cache = 0,
     using_video_margins = false,
+    border = true,
```

Comment:
I assume the border observer is always called before `window_controls_enabled()` is called for the first time? otherwise `auto` might be buggy if it starts without a border but `state.border` is true.

Reply:
Empirically, it seems to work as we intend.